specs/DefaultDownloadDialog.md

```diff
@@ -0,0 +1,208 @@
+        offset = new System.Drawing.Point(padding, padding);
+    }
+    else if (position == "Top-right")
+    {
+        newPosition =
+            CoreWebView2DefaultDownloadDialogPosition.TopRight;
+        offset = new System.Drawing.Point(-padding, padding);
+    }
+    else if (position == "Bottom-left")
+    {
+        newPosition =
+            CoreWebView2DefaultDownloadDialogPosition.BottomLeft;
+        offset = new System.Drawing.Point(padding, -padding);
+    }
+    else if (position == "Bottom-right")
+    {
+        newPosition =
+            CoreWebView2DefaultDownloadDialogPosition.BottomRight;
+        offset = new System.Drawing.Point(-padding, -padding);
+    }
+    webView.CoreWebView2.SetDefaultDownloadDialogPosition(
+        newPosition, offset);
+}
+```
+
+# API Details
+
+
+```c#
+/// The default download dialog can be aligned to any of the WebView corners.
+/// Use `SetDefaultDownloadDialogPosition` to specify a position and optional
+/// offset from that position.
+typedef enum COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION {
+
+  /// Top-left corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_TOP_LEFT,
+
+  /// Top-right corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_TOP_RIGHT,
+
+  /// Bottom-left corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_BOTTOM_LEFT,
+
+  /// Bottom-right corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_BOTTOM_RIGHT,
+
+} COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION;
+
+[uuid(9139c04d-8f37-42ae-8b63-01940c34d22f), object, pointer_default(unique)]
+interface ICoreWebView2_6 : ICoreWebView2_5
+{
+  /// `TRUE` if the default download dialog is currently open.
+  [propget] HRESULT IsDefaultDownloadDialogOpen(
+      [out, retval] BOOL* isDefaultDownloadDialogOpen);
+
+  /// Open the default download dialog. If the dialog is opened before there
+  /// are recent downloads, the dialog shows all past downloads for the
+  /// current profile. Otherwise, the dialog shows only the recent downloads
+  /// with a "See more" button for past downloads.
+  ///
+  /// \snippet ViewComponent.cpp ToggleDefaultDownloadDialog
+  HRESULT OpenDefaultDownloadDialog();
+
+  /// Close the default download dialog.
+  HRESULT CloseDefaultDownloadDialog();
+
+  /// Set the default download dialog position relative to the WebView bounds.
+  /// The dialog can be positioned against any of the WebView corners (see
+  /// COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION). When the WebView or dialog
+  /// changes size, the dialog keeps its position relative to the corner.
+  ///
+  /// The offset is an (x, y) coordinate in physical pixels that applies to the
+  /// dialog corner nearest to the COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION
+  /// specified. Use (0, 0) if no offset should be applied.
+  ///
+  /// The default position is top-right corner with offset (-30, 10). The dialog
```

Comment:
We probably want to leave ourselves room for the download UX folks to change the size and position of the download dialog. That is, they  may want to change the width or the height per download and so on. Is it a problem if we don't document those sizes here?